Source/WTF/wtf/text/CStringView.h

```diff
@@ -70,17 +70,15 @@ class CStringView final {
         m_spanWithNullTerminator = byteCast<char8_t>(literal.spanIncludingNullTerminator());
     }
 
-    unsigned hash() const;
     bool isNull() const { return m_spanWithNullTerminator.empty(); }
 
     // This method is designed to interface with external C functions handling UTF8 strings. Interactions with other
     // strings should be done through String with the exception of ASCIILiteral because ASCII is also UTF8.
     const char* utf8() const LIFETIME_BOUND { return reinterpret_cast<const char*>(m_spanWithNullTerminator.data()); }
-    size_t length() const { return m_spanWithNullTerminator.size() > 0 ? m_spanWithNullTerminator.size() - 1 : 0; }
-    std::span<const char8_t> span8() const LIFETIME_BOUND { return m_spanWithNullTerminator.first(length()); }
+    size_t sizeInBytes() const { return m_spanWithNullTerminator.size() > 0 ? m_spanWithNullTerminator.size() - 1 : 0; }
```

Comment:
This is not a better name. For a null-terminated string, length clearly means “not including the null terminator”, and size is ambiguous but seems to mean “including the null terminator”. I suggest we don’t make this name change.

Reply:
I agree, `lengthInBytes` or `byteLength` would be better than `length`.